lib/mayaUsd/ufe/UsdTransform3dPointInstance.cpp

```diff
@@ -137,6 +149,29 @@ Ufe::Matrix4d UsdTransform3dPointInstance::segmentExclusiveMatrix() const
     return UsdTransform3dBase::segmentInclusiveMatrix();
 }
 
+bool UsdTransform3dPointInstance::isAttributeEditAllowed(const PXR_NS::UsdAttribute& attr) const
+{
```

Comment:
Point instancer Transform3d implementation does not derived from the shared base class, so it needed similar code.